### DISCUSSION TITLE
نیاز به API

### DISCUSSION THREAD

**Discussion by SobhanMosavi** [mosishon/ferenzi#3] [Q&A] [answered]
سلام این پروژه حتما نیاز به API key تلگرام داره؟

**Comment by mosishon** [comment, marked as answer]
سلام بله ، شما حتما باید کلید های مورد نیاز رو از تلگرام دریافت کنید و قرار بدید تا ربات بتونه به نحو احسنت کار کنه.

**Reply by SobhanMosavi** [reply]
ممنونم ازتون
